snap/snapcraft.yaml

```diff
@@ -327,7 +327,7 @@ plugs:
   jaas-plugin:
     interface: content
     content: jaas-plugin
-    target: $SNAP/usr/bin
+    target: $SNAP/jaas/bin
```

Comment:
Has this path always been the same? Shouldn't it respect unix-like directories (usr, usr/share, usr/bin, opt, etc..)?

Reply:
It's been the same since I introduced it in https://github.com/juju/juju/pull/16956.
But it's been incorrect since then but noticed now.

Reply:
Wrt Unix-like directories I'm not sure what would be better, any ideas?

Reply:
what do you think about `$SNAP/opt/jaas/bin` for example?

Reply:
That is more aligned with unix conventions but it conflicts with the `microk8s` content-interface a few lines up which mounts the folder at `$SNAP_DATA/microk8s`. That one is just data so it's going into `$SNAP_DATA`. Not sure if that changes your opinion. If not, I'm happy to go with `$SNAP/opt/jaas/bin`.

Reply:
I think $SNAP_DATA is read/write, so maybe not the best place for bins (not sure what microk8s stores there). See https://snapcraft.io/docs/environment-variables#heading--snap-data.

Reply:
I prefer `$SNAP/opt/jaas/bin` in that case.

Reply:
Agreed, I meant more that if we use `opt/jaas/bin` maybe that's a bit confusing compared to just `<something>/jaas`. Just a nit as well. I think `$SNAP/opt/jaas/bin` is fine.

Reply:
I'll change it so long.